cpp/tests/tests.h

```diff
@@ -87,8 +90,12 @@ namespace Tests {
   //testmisc.cpp
   void runCollectFilesTests();
   void runLoadModelTests();
+
   //testpassing.cpp
   void runPassingTests();
+
+  //testbook.cpp
+  void runBookTests();
```

Comment:
I suggest that we move our own tests to the bottom of the file:
```suggestion
  //testbook.cpp
  void runBookTests();

  // testpassing.cpp (KataGo-custom test)
  void runPassingTests();
```